File: src/PrecizeSoft.GetPdfOnline.Web.MvcCoreApp/Models/Statistics.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using PrecizeSoft.GetPdfOnline.Domain.Models;
using PrecizeSoft.IO.Contracts.ConversionStatistics;

namespace PrecizeSoft.GetPdfOnline.Web.MvcCoreApp.Models
{
    public class Statistics
    {
        public ISummaryStat Summary { get; set; }

        public IEnumerable<IStatByFileCategory> StatByFileCategories { get; set; }

        public IEnumerable<IStatByHour> DailyStat { get; set; }
    }
}

File: src/PrecizeSoft.GetPdfOnline.Web.NgApp/src/app/statistics/shared/statistics.ts
import { SummaryStat } from './summary-stat';
import { StatByFileCategory } from './stat-by-file-category';
import { StatByHour } from './stat-by-hour';

export class Statistics {
    summary: SummaryStat;
    statByFileCategories: StatByFileCategory[];
    dailyStat: StatByHour[];
}
